rootfs/usr/bin/adsbhub.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# ------------------------------------------------------------------
+# www.adsbhub.org
+# version: 1.06
+# ------------------------------------------------------------------
+
+ckey="${CLIENTKEY}"
+#cmd="nc -w 60 -q 10 ${BEASTHOST} ${BEASTPORT} | nc -w 60 -q 10 data.adsbhub.org 5001"
+cmd="socat TCP4:${BEASTHOST}:${BEASTPORT} TCP4:data.adsbhub.org:5001"
+#cmd="nc -w 60 -q 10 localhost 30002 | nc -w 60 -q 10 94.130.23.233 5001"
```

Comment:
Original command has:

* `-w 60` - connection timeout after 60 seconds
* `-q 10` - sleep 10 seconds after connection ends

Presumably these are there for a reason, so if we're going to use `socat`, we should try and do the same thing.

I _think_ we want `-t 10 -T 60` added to the `socat` command (before the existing arguments).

Also, for security, we probably want to leverage `socat`'s unidirectional mode, so I'd also add `-u`, that way data can only go from local to ADSBHub (and not the other direction).